app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'sinatra/partial'
+require './lib/peep'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+    register Sinatra::Partial
+  end
+
+  enable :sessions
+
+  get '/' do
+    @peeps = Peep.all
+    @peep_time = session[:time]
+    p @peep_time
+    erb(:index)
+  end
+    
+  post '/peep' do
+    @peep = Peep.add(content: params[:peep])
+    session[:time] = @peep.time
```

Comment:
Are times being stored in the database, or just held in sessions? What would happen if the server went down?